package.json

```diff
@@ -154,6 +154,11 @@
         "category": "LaTeX Workshop"
       },
       {
+        "command": "latex-workshop.binaries",
```

Comment:
@James-Yu perhaps we should change this to `latex-workshop.validate_binaries` or something to make it clear we are checking something?